plugins/tor/tor_

```diff
@@ -0,0 +1,491 @@
+#!/usr/bin/python2
+
+from __future__ import print_function
+import collections
+import json
+import os
+import stem
```

Comment:
Please group non-core modules (here: `stem`) separately (according to [PEP8](https://www.python.org/dev/peps/pep-0008/)).

Additionally you should mention this dependency in the documentation header.

Please also note, that missing external dependencies need to handled gracefully: the should be reported when running `autoconf`.